ichnaea/alembic/versions/000000000000_base.py

```diff
@@ -13,10 +13,7 @@
 import sqlalchemy as sa
 from sqlalchemy import text
 
-from ichnaea.config import (
-    DB_RO_URI,
-    DB_RW_URI,
-)
+from ichnaea.conf import settings
```

Comment:
This reminds me of Django. I don't know if there are better models for this. I like `settings` because that's exclusively used in Ichnaea to mean "settings". I put it in `ichnaea.conf` because I had to put it somewhere. I'm game for other ideas.